README.md

```diff
@@ -1,121 +1,181 @@
    - `buzzard_high-res` (higher resolution, smaller sky area)
    - `buzzard_v1.6_1`, `buzzard_v1.6_2`, `buzzard_v1.6_3`, `buzzard_v1.6_5`, `buzzard_v1.6_21` (different realizations of `buzzard`)
 
-3. DC2 catalogs:
-   - `dc2_coadd_run1.1p`: "coadd catalog" for Run 1.1p
+3. DC2 "Coadd Catalogs":
+   by LSST DESC, compiled by Michael Wood-Vasey
+   - `dc2_coadd_run1.1p`: coadd catalog for Run 1.1p
    - `dc2_coadd_run1.1p_tract4850`: same as `dc2_coadd_run1.1p` but has only one tract (4850) for testing purpose / faster access
-   - `dc2_reference_run1.1p`: "reference catalog" for Run 1.1p
-   - `dc2_reference_run1.2p`: "reference catalog" for Run 1.2p
-   - `dc2_instance_example1`: an example instance catalog
 
-4. DC1 catalog: 
-   - `dc1`
+4. DC2 "Reference Catalogs":
+   by LSST DESC, compiled by Scott Daniel
+   - `dc2_reference_run1.1p`: reference catalog for Run 1.1p
+   - `dc2_reference_run1.2p`: reference catalog for Run 1.2p
 
-Each of the catalogs is specified by a YAML config file, which can be found [here](GCRCatalogs/catalog_configs). The galaxy quantities in these catalogs conform to [the schema](GCRCatalogs/SCHEMA.md).
+5. DC2 "Instance Catalogs":
+   by LSST DESC, compiled by Scott Daniel
+   - `dc2_instance_example1`: an example instance catalog
+   - `dc2_instance_example2`: another example instance catalog
 
+6. DC1 Galaxy Catalog:
+   - `dc1`: Galaxy catalog used for DC1 (also known as "the catalog on fatboy")
 
-## Use GCRCatalogs on NERSC
 
-_Note_: These instructions about Python environment may change in the future. If you encounter issues, please check if there's any updates on these instructions.
+## Use GCRCatalogs at NERSC
 
-`GCRCatalogs` is already installed in the DESCQA Python envoirnment at NERSC. To use it:
+Here's the instruction of using `GCRCatalogs` at NERSC.
+All catalogs that are available in `GCRCatalogs` are all physically located at NERSC.
+Note that you need to be in the `lsst` user group to access them.
+You can find instructions about getting NERSC account and joining `lsst` group
+at [this Confluence page](https://confluence.slac.stanford.edu/x/mgRTD)
+(*DESC members only*).
 
-### In Jypeter notebooks:
+### With Jypeter notebooks:
 
-First, [start a NERSC notebook server](https://jupyter-dev.nersc.gov) and open a notebook with a Python kernel. In the first cell, insert the Python enviornment to `sys.path`:
+It is recommended that you first install DESC-specific kernels for your
+NERSC jupyter-dev environment (*you only need to do this once*).
+To do so, log in to cori.nersc.gov and run:
+```bash
+source /global/common/software/lsst/common/miniconda/kernels/setup.sh
+```
+Detailed instructions can also be found at [this Confluence page](https://confluence.slac.stanford.edu/x/1_ubDQ)
+(*DESC members only*).
+
+Then, you can [start a NERSC notebook server](https://jupyter-dev.nersc.gov)
+and open a notebook with the `desc-python` or `desc-stack` kernel.
```

Comment:
@yymao [Off topic]: Should we train ourselves to use these specific names for the kernels when preparing the Notebook tutorials?  The default kernel gets saved in the Notebook and it was be an easing convenience to the user to not flag up a box making them select a kernel.

Reply:
Yes, @heather999 and I are working on a more convenient way of setting up these kernels and now if people follow the one-line instruction, these kernels will be called `desc-python` and `desc-stack`. So I think we should use these kernel names in the tutorial notebooks.